### PR TITLE
feat: help popup support vim-like key bindings for navigation

### PR DESCRIPTION
**Pull request by wugeer**
It changes the following:
- help popup support vim-like key bindings for navigation

I followed the checklist:
- [ ] I added unittests
- [x] I ran `make check` without errors
- [ ] I tested the overall application
- [x] I added an appropriate item to the changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 * add popups for viewing, adding, updating and removing remotes [[@robin-thoene](https://github.com/robin-thoene)] ([#2172](https://github.com/extrawurst/gitui/issues/2172))
++ help popup support vim-like key bindings for navigation[[@wugeer](https://github.com/wugeer)]
```
Comment (extrawurst) [inline CHANGELOG.md:19]: please check other non-issue PRs and how they are linked in the changelog

File: src/keys/key_list.rs [hunk 2]
```diff
@@ -156,7 +158,9 @@ impl Default for KeysList {
 			home: GituiKeyEvent::new(KeyCode::Home,  KeyModifiers::empty()),
 			end: GituiKeyEvent::new(KeyCode::End,  KeyModifiers::empty()),
 			move_up: GituiKeyEvent::new(KeyCode::Up,  KeyModifiers::empty()),
+			vim_like_move_up: GituiKeyEvent::new(KeyCode::Char('k'),  KeyModifiers::empty()),
 			move_down: GituiKeyEvent::new(KeyCode::Down,  KeyModifiers::empty()),
+			vim_like_move_down: GituiKeyEvent::new(KeyCode::Char('j'),  KeyModifiers::empty()),
```
Comment (extrawurst) [inline src/keys/key_list.rs:163]: why do we need a new keybinding for this? why does the `move_down` not work?

**Review by extrawurst — COMMENTED** [review]
I doubt we need this. i dont see why rebinding the normal `move_down`/up wont work in the help dialog?

**Comment by extrawurst** [comment]
Did you try using the config file we provide for vim keys: https://github.com/extrawurst/gitui/blob/master/vim_style_key_config.ron

**Comment by wugeer** [comment]
Thx!

**Pull request closed without merging**